builtin/providers/azurerm/resource_arm_storage_container.go

```diff
@@ -47,6 +49,24 @@ func resourceArmStorageContainer() *schema.Resource {
 	}
 }
 
+func validateArmStorageContainerName(v interface{}, k string) (ws []string, errors []error) {
```

Comment:
Can we put a link to the relevant document in a comment here so we can refer to it easily in future?